src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -92,14 +92,21 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             }
 
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $this->canAccess($this->resourceAccessChecker, $resourceMetadata, $resourceClass, $info, $item, $operationName);
+            if($operationName === 'update' || $operationName === 'delete') {
```

Comment:
Should be `'create' !== $operationName` (the security rule must be triggered for custom mutations).

Reply:
Absolutely, good catch. Will update tomorrow :)